demo/index.js

```diff
@@ -1,11 +1,11 @@
 import { loadPlugin, HTMLClip } from "@donkeyclip/motorcortex";
 import Player from "@donkeyclip/motorcortex-player";
-import MotorCortexGraph from "../dist/motorcortex-graphs.esm";
+import MotorCortexGraph from "../src";
```

Comment:
this will cause webpack to compile src as well which means that it will create a different bundle than the production one, why are we doing this?

Reply:
just wanted to check smthng, on it